l10n_nl_cbs_export/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
I don't see any group with creation rights for model cbs.export.file.
At least group "Account Advisor" must have creation/write rights.